#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_m68k_workaround.dpatch by Junichi Uekawa <dancer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: m68k gcc ICE's with -O3, reduce to -O2. 
## DP: closes: #328285

@DPATCH@
diff -urNad ladspa-sdk-1.1~/src/makefile ladspa-sdk-1.1/src/makefile
--- ladspa-sdk-1.1~/src/makefile	2005-09-15 06:57:26.995816288 +0900
+++ ladspa-sdk-1.1/src/makefile	2005-09-15 06:57:39.307944560 +0900
@@ -16,7 +16,7 @@
 
 INCLUDES	=	-I.
 LIBRARIES	=	-ldl -lm
-CFLAGS		=	$(INCLUDES) -O3 -fPIC
+CFLAGS		=	$(INCLUDES) -O2 -fPIC
 CXXFLAGS	=	$(CFLAGS)
 PLUGINS		=	../plugins/amp.so				\
 			../plugins/delay.so				\
